core/src/test/java/io/onetable/iceberg/TestIcebergDataHelper.java

```diff
@@ -255,14 +280,11 @@ private Object generateRandomValueForType(
         return LocalDate.ofEpochDay(randomDay);
       case TIME:
         long totalMicrosInDay = ChronoUnit.DAYS.getDuration().toMillis() * 1000;
-        long randomTimeInMicros = ThreadLocalRandom.current().nextLong(totalMicrosInDay);
-        return randomTimeInMicros;
+        return ThreadLocalRandom.current().nextLong(totalMicrosInDay);
```

Comment:
Should we just use `RANDOM` here instead of ThreadLocalRandom?

Reply:
It doesn't have method to generate long in range, now moved to generating double and used multiplier.